projects/admin/src/app/record/brief-view/items-brief-view/items-brief-view.component.html

```diff
@@ -38,5 +37,9 @@ <h5 class="mb-0 card-title">
         {{ record.metadata.barcode }}
       </dd>
     </ng-container>
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Pending count</dt>
```

Comment:
I'd prefer "Current pending requests" for label. If it is too long, "Current requests" is better than "pending count".

Reply:
Same thing here and to respect the wording of the sorting menu.